locales/en-US/webextension.properties

```diff
@@ -1,9 +1,10 @@
-addonDescription = Firefox Screenshots takes clips and screenshots from pages, and can save a permanent copy of a page.
-addonAuthorsList = Ian Bicking, Donovan Preston, and Bram Pitoyo
+addonDescription = Firefox Screenshots takes clips and screenshots from the Web and lets you save them temporarily or permanently.
+addonAuthorsList = Ian Bicking, Donovan Preston, Jared Hirsch, Danny Coates, Morpheus Chen, Helen Huang, Mark Liang, Fang Shih, John Gruen, Bram Pitoyo, Sharon Bautista and Sean Martell, Michelle Heubusch
 toolbarButtonLabel = Take a shot
-contextMenuLabel = Create screenshot
+contextMenuLabel = Take a Screenshot
 myShotsLink = My Shots
-screenshotInstructions = Drag or click on the page to select a region. Press ESC to cancel.
+screenshotInstructionOne = Drag or click on the page to select a region.
+screenshotIntrouctionTwo = Press ESC to cancel.
```

Comment:
Oh also, the template that renders this string will have to get updated to include both keys.